src/avfoundation.cs

```diff
@@ -13513,7 +13513,7 @@ interface AVCompositionTrackFormatDescriptionReplacement : NSSecureCoding {
 		CMFormatDescription ReplacementFormatDescription { get; }
 	}
 
-	delegate /* OSStatus */ int AVAudioSourceNodeRenderHandler (bool isSilence, AudioTimeStamp timestamp, uint frameCount, ref AudioBuffers outputData);
+	delegate /* OSStatus */ int AVAudioSourceNodeRenderHandler (ref bool isSilence, ref AudioTimeStamp timestamp, uint frameCount, ref AudioBuffers outputData);
```

Comment:
In the header, timestamp is passed as a ref and also const. Is there a way to make the param both ref and const here? I tried `ref const` and `const ref` and both failed to compile.

Reply:
API breaks :/